src/components/pages/registration/skills.js

```diff
@@ -19,36 +22,30 @@ class RegistrationSkills extends React.Component {
 
           <Form>
             <Form.Group>
-              <Form.Check>
+              <Form.Check className="checkbox-wrapper">
                 <Form.Check.Label>Do you have a car and can drive ?</Form.Check.Label>
                 <Form.Check.Input type="checkbox" name="driver"/>
               </Form.Check>
-              <Form.Check>
+              <Form.Check className="checkbox-wrapper">
                 <Form.Check.Label>Can you pick up stuff and leave it ?</Form.Check.Label>
                 <Form.Check.Input type="checkbox" name="picker"/>
               </Form.Check>
-              <Form.Check>
+              <Form.Check className="checkbox-wrapper">
                 <Form.Check.Label>Can you do shopping ?</Form.Check.Label>
                 <Form.Check.Input type="checkbox" name="shopper"/>
               </Form.Check>
-              <Form.Check>
+              <Form.Check className="checkbox-wrapper">
                 <Form.Check.Label>Will you travel by bicycle or walk ?</Form.Check.Label>
                 <Form.Check.Input type="checkbox" name="walker"/>
               </Form.Check>
-              <Form.Check>
+              <Form.Check className="checkbox-wrapper">
                 <Form.Check.Label>Can you help with home work ?</Form.Check.Label>
                 <Form.Check.Input type="checkbox" name="artist"/>
               </Form.Check>
-              <Form.Check>
+              <Form.Check className="checkbox-wrapper">
```

Comment:
@tombjarne when I display the page, I don't see the new styling on this page. Am I missing something ?

Reply:
@santiagoarriaga you might need to install the dependencies for the Pagination in React Bootstrap. It actually should have updated the package.json as well with the new dependency.

Reply:
@tombjarne also, the step styling looks much uglier in my browser than in the capture you had, are some styles missing in the branch ?

Reply:
@santiagoarriaga Ah right! The App.css changes were not taken over. I just pushed it to the current branch! Sorry I missed out on that one

Reply:
@tombjarne with latest update now looks good, thanks !